src/sass/mui/_select.scss

```diff
@@ -117,4 +117,16 @@ $xFormLabelLineHeight: floor($mui-label-font-size * 1.25);
       background-color: mui-color('grey', '200');
     }
   }
+
+  > .optgroup-option {
+    text-indent: 1em;
+  }
+
+  > .optgroup-label {
```

Comment:
What do you think about decreasing the optgroup label font-size? Here's how it looks at 14px:
![devbox_8080_examples_css-js_select_html](https://cloud.githubusercontent.com/assets/75881/18037893/43104e66-6d5c-11e6-8bc1-602f64a1de0c.png)